contracts/Main.sol

```diff
@@ -100,12 +133,22 @@ contract Main {
         _;
     }
 
-    function setAdmin(address _admin) public onlyAdmin {
+    function setAdmin(address _admin) external onlyAdmin {
         require(
             _admin != address(0),
             "shouldn't set admin to the zero address"
         );
         admin = _admin;
         emit NewAdmin(_admin);
     }
+
+    function rescueToken(address _token) external onlyAdmin {
```

Comment:
Is this in case there are uniswap token rewards?

Reply:
Yes! But also, if the contract needs to call a function to claim, we probably won't be able to get it

Reply:
When I claimed mine I remember I had to call a function.